tests/unit/lib/geo-location.test.js

```diff
@@ -116,3 +116,37 @@ test('`getGeoLocation` returns mock geolocation data if `mode: "mock"`', async (
   t.false(hasCalledStateSet)
   t.deepEqual(geo, mockLocation)
 })
+
+test('`getGeoLocation` returns mock geolocation data if valid country code set', async (t) => {
+  const returnedLocation = {
+    city: 'Netlitown',
+    country: { code: 'CA', name: 'Mock Country' },
```

Comment:
It would, but I was advised not to create the necessary file to do this lookup as the data could change, and there are numerous subdivisions in each country etc.  Any testing that needed to be done could be done by country code.  If there is a demand for it, I think we could easily add a file with a map of codes => names though.